### DISCUSSION TITLE
Which version has argo cp?

### DISCUSSION THREAD

**Discussion by rafaela-soares** [argoproj/argo-workflows#9327] [General]
Hello,

I was trying to use `argo cp` but the command seems to not exist.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/74001161/183857077-88fd8f10-6aaa-421e-b5f7-85c0d9bde1ee.png">

**Comment by kostis-codefresh** [comment]
Hello

It is part of the RC version https://github.com/argoproj/argo-workflows/releases/download/v3.4.0-rc1/argo-linux-amd64.gz
